recommenders/__init__.py

```diff
@@ -1,33 +1,34 @@
-import pandas as pd
-import numpy as np
-import os
-
-#print(os.getcwd())
-
-default_poster_url = "https://gurukul-be.s3.ap-south-1.amazonaws.com/0c3df9f4dc56471d88bb88255bb79548.jpg"
-
-####################################################################################################################################
```

Comment:
The line consists of several hash signs (#) which are unnecessary and can be removed.